visualizations/frontend/ResetAll.vue

```diff
@@ -0,0 +1,19 @@
+<template>
+  <button @click="handleClick">{{ text }}</button>
```

Comment:
I deliberately didn't style this. 

I feel like we need a 'Link as button' component that was the styling, and we use that inside a component that takes care of handling the click. So I went with a proof of concept. This should be replaced with a nice looking component.